yarn-project/foundation/src/log/logger.ts

```diff
@@ -15,6 +13,9 @@ export type LogLevel = (typeof LogLevels)[number];
 const envLogLevel = process.env.LOG_LEVEL?.toLowerCase() as LogLevel;
 const currentLevel = LogLevels.includes(envLogLevel) ? envLogLevel : DefaultLogLevel;
 
+const namespaces = process.env.DEBUG ?? 'aztec:*,-json-rpc:*';
```

Comment:
Sane default here, but can be overridden by the environment

Reply:
oh thanks thanks thanks thanks